classes/items/itembase.cpp

```diff
@@ -18,44 +18,62 @@ void Item::generate()
 
 
 //Constructors / Destructors
-Item::Item()
-{
+Item::Item(std::string name,
+            unsigned int type,
+            unsigned int tier,
+            unsigned int value,
+            unsigned int level);
+
     std::string name;
-    unsigned type;
-    unsigned value;
+    unsigned int type;
+    unsigned int value;
+    unsigned int tier;
+    unsigned int level;
```

Comment:
Remove these lines as you're re-stating the variables in the constructor which isn't allowed(I think that'll fix that error)